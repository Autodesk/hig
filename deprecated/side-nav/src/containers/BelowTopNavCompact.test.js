import React from "react";
import renderer from "react-test-renderer";

import BelowTopNavCompact from "./BelowTopNavCompact";

describe("side-nav/containers/BelowTopNavCompact", () => {
  describe("snapshot tests", () => {
    const cases = [
      {
        description: "renders without props",
        props: {},
      },
    ];

    cases.forEach(({ description, props: { children, ...otherProps } }) => {
      it(description, () => {
        const wrapper = (
          <BelowTopNavCompact {...otherProps}>{children}</BelowTopNavCompact>
        );
        const tree = renderer.create(wrapper).toJSON();

        expect(tree).toMatchSnapshot();
      });
    });
  });
});
