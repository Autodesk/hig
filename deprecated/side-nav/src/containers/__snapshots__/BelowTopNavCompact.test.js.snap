// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`side-nav/containers/BelowTopNavCompact snapshot tests renders without props 1`] = `
.emotion-0 {
  background-color: #ffffff;
  padding: 0 0;
  border-radius: 0;
}

<div
  className="emotion-0"
/>
`;
